porcupine/plugins/text_google_search.py

```diff
@@ -0,0 +1,23 @@
+"""Search selected text on Google."""
+
+import tkinter as tk
+import webbrowser
+import urllib.parse
+
+from porcupine import menubar, tabs
+
+def google_search(tab: tabs.FileTab) -> None:
+    try:
+        selected_text = tab.textwidget.get("sel.first", "sel.last")
+    except tk.TclError:
+        # nothing selected
+        return
+    
+    # Check multi line or text with only spaces
+    if selected_text.strip() and "\n" not in selected_text and not selected_text.isspace():
```

Comment:
I don't think you need both `.strip()` and `.isspace()`. One of the two is enough.

Reply:
```suggestion
    if "\n" not in selected_text and not selected_text.isspace():
```

Reply:
.strip() kept .isspace() deleted.